content/documentation/staging/features/01-istio-service-mesh-overview.adoc

```diff
@@ -0,0 +1,14 @@
+---
+title: "Service Mesh Overview"
+date: 2021-07-26T00:00:00+02:00
+draft: false
+weight: 1
```

Comment:
I think that as long as you have the weight, you don't need to put the numerics index as a prefix int the file name.
The file name in hugo is used to shape the URL of the page. IMO, for better search engine practices, we could do without the prefix.

Reply:
ah, right, I wonder if I can modify the url but not the file, I put the numeric index in the file to keep them ordered in the folder, but yes, the url is not needed, it's more to not get lost when working with long files.
I'll search if both aspects can be met.